File: Shared/Enums/ExecutionStatus.cs
namespace Shared.Enums
{
    public enum ExecutionStatus
    {
        NoStatus = 0,
        Succeded = 1,
        CompileError = 2,
        RuntimeError = 3,
        TimedOut = 4,
        Cancelled = 5,
        FailedToExecute = 6,
        Pending = 7
    }
}

File: Runners/JavaRunner/java-runner/src/main/java/com/mems/Shared/Enums/ExecutionStatus.java
package com.mems.Shared.Enums;

import com.fasterxml.jackson.annotation.JsonValue;

public enum ExecutionStatus {
    NO_STATUS(0),
    SUCCEEDED(1),
    COMPILE_ERROR(2),
    RUNTIME_ERROR(3),
    TIMED_OUT(4),
    CANCELLED(5),
    FAILED_TO_EXECUTE(6),
    PENDING(7);

    private final int value;

    ExecutionStatus(int value) {
        this.value = value;
    }

    @JsonValue
    public int getValue() {
        return value;
    }
}
